shared/src/chain_state.rs

```diff
@@ -390,13 +390,14 @@ impl ChainState {
                             return Err(e);
                         }
                         tx_pool.update_statics_for_add_tx(tx_size, cycles);
-                    } else if self
-                        .pending_tx(&mut tx_pool, Some(cycles), tx_size, tx)
-                        .is_ok()
-                    {
+                        return Ok(cycles);
```

Comment:
Can we use `else if let Ok(cycles) {....}` to eliminate `return`? it's more readability

Reply:
It's on purpose. If using `else if`, there will be more nesting conditions, lead the code to complicated. Return result for every branch.